### PR TITLE
feat(api): Allow owners and managers to add and remove members to/from teams

### PR DESCRIPTION
**Pull request by macqueen**
they could do this from the org member details endpoint, but not this one

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

File: src/sentry/api/endpoints/organization_member_team_details.py [hunk 1]
```diff
@@ -44,8 +45,7 @@ class RelaxedOrganizationPermission(OrganizationPermission):
 class OrganizationMemberTeamDetailsEndpoint(OrganizationEndpoint):
     permission_classes = [RelaxedOrganizationPermission]
 
-    def _can_access(self, request, member):
-        # TODO(dcramer): ideally org owners/admins could perform these actions
```
Comment (MaxBittker) [inline src/sentry/api/endpoints/organization_member_team_details.py]: nice

File: tests/sentry/api/endpoints/test_organization_member_team_details.py [hunk 2]
```diff
@@ -111,6 +111,189 @@ def test_can_join_as_member_with_open_membership(self):
...
+        resp = self.client.post(path)
+
+        assert resp.status_code == 201
+
+        assert OrganizationMemberTeam.objects.filter(
+            team=team,
+            organizationmember=member_om,
+        ).exists()
+
+    def test_manager_cannot_add_owner(self):
```
Comment (MaxBittker) [inline tests/sentry/api/endpoints/test_organization_member_team_details.py]: do we want to enforce this specific part? seems harmless

**Review by MaxBittker — APPROVED** [review]
the code and tests look good to me - but IMO we could go further with letting admins manage owners - maybe a future PR?

File: tests/sentry/api/endpoints/test_organization_member_team_details.py (hunk 2 above)
Comment (macqueen) [reply tests/sentry/api/endpoints/test_organization_member_team_details.py]: i just wanted to keep it as close to the current permissions as possible for now

**Pull request merged**